guides/front_end/react_quick_start.mdx

```diff
@@ -76,7 +76,7 @@ Import the CSS library to style the search components.
 import React from 'react';
 import { InstantSearch, SearchBox, InfiniteHits } from 'react-instantsearch';
 import { instantMeiliSearch } from '@meilisearch/instant-meilisearch';
-import 'instantsearch.css/themes/algolia.css';
+import 'instantsearch.css/themes/satellite.css';
```

Comment:
will this affect the final screenshot too?

Reply:
It does, I've just updated it